dockerfiles/Dockerfile.ci.es

```diff
@@ -0,0 +1,75 @@
+FROM debian:bookworm-slim AS base
+
+ARG STAC_FASTAPI_TITLE
```

Comment:
This is setting all values not specified in the `.env` file to empty strings. Could you modify to handle default values?